src/main/java/dumpling/command/Command.java

```diff
@@ -0,0 +1,24 @@
+package dumpling.command;
```

Comment:
Well done on abstracting wherever you could. I really like how all your classses are organized and abstracted into packages!